sdk/lib/drivers/wdf/kmdf/src/librarycommon/fxlibrarycommon.cpp

```diff
@@ -480,6 +499,12 @@ FxLibraryCommonRegisterClient(
             __Print((LITERAL(WDF_LIBRARY_REGISTER_CLIENT)
                      ": Enhanced Verification is ON \n"));
 
+            __Print((LITERAL(WDF_LIBRARY_REGISTER_CLIENT)
+                     "Verificated functions table NOT IMPLEMENTED \n"));
+
+            __debugbreak();
```

Comment:
What is that?

Reply:
Driver has two function tables - VfWdfVersion and WdfVersion. VfWdfVersion should be edit. Thats why added message. Debugbreak can be replace with return status INVALID_PARAMETER. Tables defined in fxdynamics.h and vffxdynamics.h

Reply:
This is a debug break for unimplemented code path (see the added debug message)